JUnit version 4.12
.......................................................................................................................................................................E.................
Time: 1.249
There was 1 failure:
1) test058(com.google.javascript.rhino.JSDocInfo_ESTest)
java.lang.NullPointerException
	at com.google.javascript.rhino.JSTypeExpression.equals(JSTypeExpression.java:107)
	at java.util.ArrayList.indexOf(ArrayList.java:321)
	at java.util.ArrayList.contains(ArrayList.java:304)
	at com.google.javascript.rhino.JSDocInfo.addImplementedInterface(JSDocInfo.java:1233)
	at com.google.javascript.rhino.JSDocInfo_ESTest.test058(JSDocInfo_ESTest.java:545)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

FAILURES!!!
Tests run: 184,  Failures: 1

